src/main/java/org/broadinstitute/hellbender/tools/copynumber/AnnotateIntervals.java

```diff
@@ -44,14 +45,14 @@
  *
  * <ul>
  *     <li>
- *         GC-content annotated-intervals file.
- *         This is a TSV with a SAM-style header containing a sequence dictionary,
- *         a row specifying the column headers contained in {@link AnnotatedIntervalCollection.AnnotatedIntervalTableColumn},
+ *         GC-content annotated-intervals file in TSV format.
+ *         This is a text-based file with tab-separated column values, a SAM-style header containing a sequence dictionary,
+ *         a row specifying the column headers, e.g. CONTIG, START, END, GC_CONTENT,
```

Comment:
@sooheelee If possible, I'd rather not include the column headers explicitly in the Javadoc here, since it's just one more place things can get out of sync if they are ever changed.  The way I originally had it, you'll see that I added extra Javadoc to AnnotatedIntervalCollection.AnnotatedIntervalTableColumn so that the headers show up there.

Unless you have strong feelings about the matter (or unless the link doesn't work properly for some reason when the html documentation is generated---I didn't check this), I'll revert.

Reply:
Also, it appears this is the only place you added the column headers?

Reply:
And finally, I think you may have modified the copy-and-pasted text "This is a TSV with a SAM-style header..." differently throughout the PR.  I'll try to make these more consistent.